app/src/main/java/com/kylecorry/trail_sense/TrailSenseApplication.kt

```diff
@@ -15,20 +15,39 @@ class TrailSenseApplication : Application(), CameraXConfig.Provider {
     override fun onCreate() {
```

Comment:
Can you undo the changes to this file?

Since all the methods are just wrapping a single line and were still easily readable before, I would prefer to not have them in separate methods

Also, the cameraXConfig change should be undone unless you can provide some reasons why it should be Camera2Config rather than CameraXConfig